sources/Metal/iTermMetalDriver.m

```diff
@@ -644,7 +644,8 @@ - (BOOL)shouldCreateIntermediateRenderPassDescriptor:(iTermMetalFrameData *)fram
     if (!_broadcastStripesRenderer.rendererDisabled && frameData.perFrameState.showBroadcastStripes) {
         return YES;
     }
-    if (!_cursorGuideRenderer.rendererDisabled && frameData.perFrameState.cursorGuideEnabled) {
+    if (!_cursorGuideRenderer.rendererDisabled && (frameData.perFrameState.cursorGuideEnabled
+                                                   || frameData.perFrameState.cursorVGuideEnabled)) {
```

Comment:
Move the `||` up to the end of the previous line for style consistency

Reply:
Done.